src/MatrixTypes.f90

```diff
@@ -946,24 +925,33 @@ SUBROUTINE pop_recv(acc,valBuf,idxBuf,ctBuf,idx,req,idxReq)
 !> @param req The array of requests for data
 !> @param idxReq The array of requests for indices
 !>
-SUBROUTINE pop_send(valBuf, idxBuf, idx, req, idxReq)
+SUBROUTINE pop_send(valBuf,idx,req,f)
```

Comment:
Update this routine's header comment as well.

Reply:
This has been fixed.
For some reason, git didn't mark this as modified when I pushed up the last commit